lib/main.ts

```diff
@@ -46,7 +72,7 @@ function getTypeScript(typescript: typeof ts) {
 }
 
 function checkAndNormalizeSettings(settings: compile.Settings): compile.Settings {
-	const { declarationFiles, noExternalResolve, sortOutput, typescript, ...standardSettings } = settings;
+	const { customTransformers, declarationFiles, noExternalResolve, sortOutput, typescript, ...standardSettings } = settings;
```

Comment:
Should this be `getCustomTransformers`?

It seems that you don't use the variable here, but we should keep it here as it will remove the property from `standardSettings`. That way we don't need the `delete settings.getCustomTransformers;` that you have later on.